src/views/EditorPage.vue

```diff
@@ -119,6 +119,8 @@ export default {
     }
   },
   mounted() {
+    this.$store.dispatch('setUser')
```

Comment:
Not sure if this is the right place to fetch user details, any better place for this @egoist?